forum/templates/forum/thread.html

```diff
@@ -18,7 +18,26 @@
 {% block leftcontents %}
     {{ block.super }}
     {% if perms.forum.change_forums %}
-        <a class="btn btn-block btn-outline-dark" href="{% url "admin:forum_thread_change" thread.id %}">Edit</a>
+        <a class="btn btn-block btn-outline-dark mb-3" href="{% url "admin:forum_thread_change" thread.id %}">Edit</a>
+    {% endif %}
+    {% if not user.is_authenticated %}
```

Comment:
does this mean if the user is not authenticated it gives them the option to do notification subscription stuff?

Reply:
fixed in 8fe7f77d7b04f055a7e50c0a8670978acd9a52d6